docs/UserGuide.md

```diff
@@ -88,11 +88,11 @@ section of this user guide or to an external website.
 
 1. Ensure you have `Java 11` or above installed in your Computer. <br><br>
 
-   * If you are using a Mac, open the terminal and enter `java --version`.
-     * To open the terminal, press `command` and `space bar` together to launch Spotlight.
+   * If you are using a Mac, open the `terminal` and enter `java --version`.
+     * To open the `terminal`, press `command` and `space bar` together to launch Spotlight.
      * Following, enter `terminal` in the search bar.
-   * If you are using a Windows, open the command window and enter `java -version`. 
-     * To open the command window, press `Windows` and `R` together to launch the "Run" box.
+   * If you are using a Windows, open the `command window` and enter `java -version`. 
+     * To open the `command window`, press `Windows` and `R` together to launch the "Run" box.
```

Comment:
Should we remove the "a" before Windows, and call it "Command Prompt"?

Reply:
Yes